src/angular-app/bellows/shared/sound-player.component.html

```diff
@@ -6,5 +6,5 @@
         {{$ctrl.currentTime()}} / {{$ctrl.duration()}}
     </span>
     <!--suppress HtmlFormInputWithoutLabel -->
-    <input class="seek-slider" type="range" min="0" max="{{$ctrl.durationInSeconds()}}" value="0">
+    <input class="seek-slider" type="range" min="0" max="{{$ctrl.durationInSeconds()}}" step="0.0000000000000001" value="0">
```

Comment:
An HTML comment about the "step" hack would be good here.

Reply:
:-D great minds!

Reply:
Yes, just added